src/main.ts

```diff
@@ -167,4 +171,60 @@ async function releasesNodeModulesCheck(repository: string, ownername: string, s
     console.log(err);
   }       
 }
+async function vulnerabilityBotCheck(repository: string, ownername: string, secret_token: string, octokit: Octokit) {
+  try{
+    const result = await octokit.request('GET /repos/{owner}/{repo}/vulnerability-alerts',{
+      repo: repository,
+      owner: ownername,
+      headers : { 
+        Authorization: 'Bearer ' + secret_token,
+      },
+      mediaType: {
+        previews: [
```

Comment:
what is the significance of this?

Reply:
This is a mandatory header. It throws this error otherwise: 
 Property 'mediaType' is missing in type '{ repo: string; owner: string; headers: { Authorization: string; }; }' but required in type '{ mediaType: { previews: ["dorian", ...string[]]; }; }'.ts(2345)
Endpoints.d.ts(20, 5): 'mediaType' is declared here.

Reply:
okay
On what basis is "dorain" value selected?

Reply:
Enabling and disabling dependency alerts for a repository using the REST API is currently available for developers to preview. To access these new endpoints during the preview period, you must provide a custom media type in the Accept header:

application/vnd.github.dorian-preview+json
☝️ This header is required.

Reply:
<img width="960" alt="2021-06-24" src="https://user-images.githubusercontent.com/84702817/123222328-77f7d380-d4ed-11eb-9cfa-7e7d9362e5c3.png">
I am using octokit na toh this is the endpoint file and isme aise hi declared hai so that's why it gives an error if I dont declare it.

Reply:
I understand that the header is mandatory
My query is regarding the header value. On what basis are we choosing that value?

Reply:
API previews let you try out new APIs and changes to existing API methods before they become part of the official GitHub API.
To access an API preview, you'll need to provide a custom media type in the Accept header for your requests. Feature documentation for each preview specifies which custom media type to provide.
The endpoint in rest api for vulnerability has dorian-preview as the custom media type. That's why we are using this value. We dont have an option to choose